packages/hardhat-core/test/fixture-projects/compilation-import-folder-from-path/contracts/A.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.19;
+
+// For test purposes this import must point to a folder
+import "../contracts";
```

Comment:
Ugh, I know I'm being a pain in the ass, but this feels too unrealistic to me. It's probably fine, but I'd rather do something like adding a `dir` directory with a `dir/B.sol` file, and import `./dir`.